### ISSUE TITLE
Element 'description': This element is not expected.

### ISSUE THREAD

**Issue by marcelloceschia** [1drop/shopware-sentry#33]
```
Unable to parse file "/tmp/5f69b07611604NDcicm.xml". 
Message: [ERROR 1871] Element 'description': This element is not expected. Expected is one of ( label, version ). (in /var/www/html/ - line 5, column 0)
```

**Comment by jhit** [comment]
I'm getting the same Error.

**Comment by bingemer** [comment]
Removed the two <description Lines in plugin.xml and was able to install it again, afterwards threw Guzzle errors on frontend. Removed the plugin again :(

**Comment by larsbo** [comment]
Same guzzle error here with Shopware 5.8.6. Broken frontend/backend/cli...
Plugin cannot be uninstalled and must be remove from file system by hand.
Maybe doing a test installation next time before releasing a totally broken version 😕

**Comment by htuscher** [comment]
It's always a big issue when wrapping third party libraries which share common libs with the Shopware core like e.g. Guzzle which is a requirement of the Sentry-SDK.
We assume that minor updates of Shopware => 5.6.7 - 5.6.8 are non breaking. It's way to much effort for an open source plugin to test every minor version.
We you're willing to contribute a github actions pipeline maybe that does that, feel free.

I will fix it shortly (if possible or mark the version incompatible).

**Comment by htuscher** [comment]
I just released 2.0.4 which has a very nasty workaround for the sentry SDK using `PrettyVersions::getVersion` to determine its own version. That depends on how the plugin and how Shopware is installed via composer. Therefore I hot-patch the PHP file from the Sentry SDK.
